create_order_flow.sh

```diff
@@ -0,0 +1,14 @@
+near call usdt.qa.v1.nearlend.testnet ft_transfer_call '{"receiver_id": "limit_orders.v1.nearlend.testnet", "amount": "10000000", "msg": "{\"Deposit\": {\"token\": \"usdt.qa.v1.nearlend.testnet\"}}"}' --accountId nearlend.testnet --depositYocto 1 --gas 300000000000000
```

Comment:
Personal scripts for testing in the main branch should not be

Reply:
this will help test it more quickly. imagine just copying Igor's flows from test documentation and trying to check whether it works or not. Plus commands are helpful